test/resources/ostree.resource

```diff
@@ -65,7 +65,6 @@ System Is Running Right Ref And Healthy
 
     Should Be Equal As Strings    ${expected_deploy}    ${current_deploy}
     Greenboot Health Check Exited
-    System Should Be Healthy
```

Comment:
I'm a little bit worried that removing this health check will allow race conditions or flakes in the test suite that will be hard to debug because an unhealthy system will rollback while the test suite is still running.

Do we need different behaviors in different test cases?

Reply:
after this change   greenboot-healthcheck will  be still expected to be successfull, it removes the extra step of checking the `health.json`  state

Reply:
"Greenboot Health Check Exited" does not check that the system it health, only that the health checks are no longer running. https://github.com/openshift/microshift/blob/main/test/resources/ostree-health.resource#L10-L20

Reply:
it expects substate of [exited](https://github.com/openshift/microshift/blob/main/test/resources/ostree-health.resource#L20)  which considers that check is succefful.

Reply:
OK, I didn't assume that "exited" meant "exited successfully"